### ISSUE TITLE
Preseed support

### ISSUE THREAD

**Issue by kyzrfranz** [canonical/microovn#14]
Hey quick question, would it be time well spent if I got --preseed support for microovn going (and then also for microceph..)?
If yes, would you have any requirements that I'd might not be aware of?
Sorry for doing this here - not quite sure where to put it on discourse.

**Comment by stgraber** [comment]
Possibly, as it stands we have so few questions that you can likely just use `microovn cluster bootstrap`, `microovn cluster add` and `microovn cluster join` directly.

**Comment by kyzrfranz** [comment]
True, but coming from an automation perspective here.
Currently doing some eval in how to automate setup and i liked the lxd approach very much.
It’s basically a matter of taste but i was thinking that as the commands already are quite similar (which makes for nice UX) - why not go all the way?

**Comment by stgraber** [comment]
yeah, the main catch is that there's no such thing as a LXD trust password here, so you'd easily be able to do a preseed to bootstrap the first server, but for any server you want to join, you'll need a join token, so you'll need to first issue a `microovn cluster add` on an existing server and then customize the YAML preseed on the joining server to include it.

Definitely possible but at that point you're getting dangerously close to just running the three commands directly ;)

**Comment by kyzrfranz** [comment]
yeah, true - i‘ll mull it over and play around with it little bit :)
